.github/workflows/publish-to-website.yml

```diff
@@ -0,0 +1,56 @@
+name: Publish to website
+
+env:
+  BRANCH_NAME: publish-latest-learning-path
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/publish-to-website.yml
+      - config/urls.yaml
+      - contributor/**
+      - introduction/**
+      - product-owner/**
+      - scripts/**
+      - trusted-committer/**
+      - workbook/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Learning Path
+        uses: actions/checkout@v2
+        with:
+          path: ./InnerSourceLearningPath
+      - name: Checkout innersourcecommons.org
+        uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.WEBSITE_DEPLOY_KEY }}
```

Comment:
This is entered in by repo admins somewhere in the UI?

Reply:
Correct. @voborgus added the public key as a Deploy Key  for https://github.com/InnerSourceCommons/innersourcecommons.org, and I added the private key to https://github.com/InnerSourceCommons/InnerSourceLearningPath as an Action secret.